ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/view/entity/DcatView.java

```diff
@@ -0,0 +1,69 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.view.entity;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.viewcreation.valueobjects.ViewName;
+import org.apache.jena.rdf.model.*;
+import org.apache.jena.vocabulary.RDF;
+
+import java.util.List;
+import java.util.Objects;
+
+import static org.apache.commons.lang3.Validate.notNull;
+import static org.apache.jena.rdf.model.ResourceFactory.createProperty;
+import static org.apache.jena.rdf.model.ResourceFactory.createResource;
+import static org.apache.jena.rdf.model.ResourceFactory.createStatement;
+
+public class DcatView {
+
+	public static final String VIEW_DESCRIPTION_SUFFIX = "/description";
+	public static final Property DCAT_DATA_SERVICE = createProperty("http://www.w3.org/ns/dcat#DataService");
```

Comment:
you COULD put "http://www.w3.org/ns/dcat#" in a separate string and then call it in each of these statements. might read prettier